### ISSUE TITLE
Shipped package has broken permissions

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [vv-vim/vv#77]
Downloading the latest version from GitHub releases, I am presented with this when attempting to open it:

<img width="372" alt="Screen Shot 2021-02-02 at 5 53 34" src="https://user-images.githubusercontent.com/200/106596705-05289480-651b-11eb-9ee4-514f936d63f6.png">

> You do not have permission to open the application "VV".
>
> Contact your computer or network administrator for assistance.

Some searching led me to [this StackExchange](https://apple.stackexchange.com/questions/408752/the-message-you-do-not-have-permission-to-open-the-application-is-shown-when-t) answer; after which I tried the following:

```sh
sudo xattr -dr com.apple.quarantine /Applications/VV.app
chmod a+x /Applications/VV.app/Contents/MacOS/VV
```

(Unfortunately, although I no longer receive the "Permissions" pop-up, I'm still unable to launch VV.app due to #76. `/=`)

**Comment by igorgladkoborodov** [comment]
Hi @ELLIOTTCABLE , thank you for reporting this. Did you try `dmg` or `zip` version? Those answers on Stack Exchange mentioned that permissions could be lost after unzipping. Also can you try `brew cask install vv`?

This looks like a Big Sur issue, I'm still on Catalina. Will update later today and try to reproduce.

**Comment by ELLIOTTCABLE** [comment]
Huh! Not only does the `.dmg` work, it also fixes #76. o_O

**Comment by igorgladkoborodov** [comment]
Could not reproduce it with zip too. Closing it. Please reopen it if it will repeat.
